Tests/js/views/assets/ez-searchview-tests.js

```diff
@@ -0,0 +1,191 @@
+                "searchRequest should have been fired after userSearchStringChange"
+            );
+        },
+
+        "Should set the limit and fire a search request": function () {
+            var searchRequestFired = false,
+                searchString = "AnySearch",
+                limit = 10,
+                loadMoreAddingNumber = 42,
+                that = this;
+
+            this.view.on('*:searchRequest', function(e) {
+                searchRequestFired = true;
+                Assert.areSame(
+                    e.searchString, searchString,
+                    "the searchRequest should have the userSearchString in param"
+                );
+                Assert.areSame(
+                    e.limit, that.view.get('limit'),
+                    "the searchRequest should have the limit in param"
+                );
+            });
+            this.view.set('searchString', 'AnySearch');
+
+            this.view.set('limit', limit);
+            this.view.set('loadMoreAddingNumber', loadMoreAddingNumber);
+
+            this.view.render();
+
+            this.view.get('searchListView').set('offset', '12');
+            this.view.fire('searchListView:offsetChange');
+
+            Assert.isTrue(
+                searchRequestFired,
+                "searchRequest should have been fired after offsetChange"
+            );
+            Assert.areSame(
+                that.view.get('limit'), limit + loadMoreAddingNumber,
+                "The limit should be updated"
+            );
+        },
+
+        "Should set the userSearchString after submitting form": function () {
+            var container = this.view.get('container');
+
+            this.view.render();
+
+            container.one('.ez-search-form-input').set('value', 'SomeResearch');
+            container.one('form').simulate('submit');
+            Assert.areSame(
+                this.view.get('userSearchString'),
+                'SomeResearch',
+                'Submitting form should set userSearchString'
+            );
+        },
+    });
+
+    searchListViewAttributeTest = new Y.Test.Case({
+        name: "eZ Search view attribute tests",
+
+        setUp: function () {
+            this.view = new Y.eZ.SearchView({});
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+        },
+
+        "Should provide the SearchListView as an attribute": function () {
+            var searchListView = this.view.get('searchListView');
+
+            Assert.isInstanceOf(
+                Y.eZ.SearchListView,
+                searchListView,
+                "The searchListView should be an instance of Y.eZ.SearchListView"
+            );
+        },
+    });
+
+    Y.Test.Runner.setName("eZ Search view tests");
+    Y.Test.Runner.add(viewTest);
+    Y.Test.Runner.add(searchListViewAttributeTest);
+
+}, '', {requires: ['test', 'event-valuechange', 'node-event-simulate', 'ez-searchview', 'ez-searchlistview']});
```

Comment:
event-valuechange not needed
also you should not include ez-searchlistview, instead you should make sure to pass a view to the SearchView or to locally defined a Y.eZ.SearchListView to really test a unit (SearchView).